org.eclipse.xtext.full.releng/pom.xml

```diff
@@ -162,4 +162,246 @@
 		<module>../org.eclipse.xtext.p2repository</module>
 	</modules>
 
+	<profiles>
+		<profile>
+			<id>sonatype-oss-release</id>
+			<properties>
+				<skipTests>true</skipTests>
+			</properties>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.sonatype.central</groupId>
+						<artifactId>central-publishing-maven-plugin</artifactId>
+						<version>0.7.0</version>
+						<extensions>true</extensions>
+						<configuration>
+							<publishingServerId>central</publishingServerId>
+							<excludeArtifacts>
+								<excludeArtifact>
+									org.eclipse.emf.mwe2.language.sdk
+								</excludeArtifact>
+								<excludeArtifact>
+									org.eclipse.xtend.core.tests
+								</excludeArtifact>
+								<excludeArtifact>
+									org.eclipse.xtend.doc
+								</excludeArtifact>
+								<excludeArtifact>
+									org.eclipse.xtend.examples
+								</excludeArtifact>
+								<excludeArtifact>
+									org.eclipse.xtend.ide
+								</excludeArtifact>
+								<excludeArtifact>
+									org.eclipse.xtend.ide.tests.data
+								</excludeArtifact>
+								<excludeArtifact>
+									org.eclipse.xtend.lib.gwt.test
+								</excludeArtifact>
+								<excludeArtifact>
+									org.eclipse.xtend.lib.tests
+								</excludeArtifact>
+								<excludeArtifact>
+									org.eclipse.xtend.m2e
+								</excludeArtifact>
+								<excludeArtifact>
+									org.eclipse.xtend.sdk
+								</excludeArtifact>
+								<excludeArtifact>
+									org.eclipse.xtend.standalone
```

Comment:
wired name @szarnekow

Reply:
IIRC, that was used long ago before the maven plug-in. It's now empty and contains only dependencies. I think it was also deprecated.